### PR TITLE
{geo}[foss/2019a] OCNet v0.2.0, SSN v1.1.14

### PR DESCRIPTION
**Pull request by arielzn**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/5e15f0a3ea8b8006e65096c891f46069 for a full test report.

File: easybuild/easyconfigs/o/OCNet/OCNet-0.2.0-foss-2019a-R-3.6.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+sources = ['%(name)s_%(version)s.tar.gz']
+checksums = ['f1c84935f89cfca4e324faa848a8ee84d0b902d6fabcc21e1354e1fff861fe01']
+
+dependencies = [
+    ('R', '3.6.0'),
+    ('SSN', '1.1.14', '-R-%(rver)s'),
+]
+
+sanity_check_paths = {
+    'files': ['%(name)s/R/%(name)s', '%(name)s/libs/%(name)s.so'],
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/o/OCNet/OCNet-0.2.0-foss-2019a-R-3.6.0.eb]:
```suggestion
    'files': ['%(name)s/R/%(name)s', '%%(name)s/libs/%%(name)s.%s' % SHLIB_EXT],
```

File: easybuild/easyconfigs/o/OCNet/OCNet-0.2.0-foss-2019a-R-3.6.0.eb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/o/OCNet/OCNet-0.2.0-foss-2019a-R-3.6.0.eb]: nitpicking: any particular reason for this?

File: easybuild/easyconfigs/s/SSN/SSN-1.1.14-foss-2019a-R-3.6.0.eb [hunk 3]
```diff
@@ -0,0 +1,36 @@
+#
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/s/SSN/SSN-1.1.14-foss-2019a-R-3.6.0.eb]: nitpicking: any particular reason for this?

File: easybuild/easyconfigs/s/SSN/SSN-1.1.14-foss-2019a-R-3.6.0.eb [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+checksums = ['32f7335fe89fe7f04d5668edfb42f9325399144b9c75fe6c07442866912e83ea']
+
+dependencies = [
+    ('R', '3.6.0'),
+    ('rgeos', '0.5-1', '-R-%(rver)s'),
+    ('rgdal', '1.4-4', '-R-%(rver)s'),
+]
+
+sanity_check_paths = {
+    'files': ['%(name)s/R/%(name)s', '%(name)s/libs/%(name)s.so'],
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/s/SSN/SSN-1.1.14-foss-2019a-R-3.6.0.eb]:
```suggestion
    'files': ['%(name)s/R/%(name)s', '%%(name)s/libs/%%(name)s.%s' % SHLIB_EXT],
```

File: easybuild/easyconfigs/s/SSN/SSN-1.1.14-foss-2019a-R-3.6.0.eb (hunk 3 above)
Comment (arielzn) [reply easybuild/easyconfigs/s/SSN/SSN-1.1.14-foss-2019a-R-3.6.0.eb]: ah, if there's a comment on the very first line of an `.eb` file, vim will detect it as "conf" filetype and it's nicely highlighted.

probably it makes more sense to leave the usual comment "This is an easybuild recipy..."

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/710b603095e46b75dace1289399fb555 for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @arielzn!

**Pull request merged**